lib/ClimaCoreSpectra/test/gcm_visual_test.jl

```diff
@@ -0,0 +1,101 @@
+# Standalone test file that tests spectra visually
+# Taken from https://github.com/CliMA/ClimateMachine.jl/blob/master/test/Common/Spectra/gcm_standalone_visual_test.jl
+using Plots
+
+using ClimaCoreSpectra:
+    compute_gaussian!,
+    compute_legendre!,
+    SpectralSphericalMesh,
+    trans_grid_to_spherical!,
+    power_spectrum_1d,
+    power_spectrum_2d,
+    compute_wave_numbers
+using FFTW
+
+
+include("spherical_helper.jl")
+
+FT = Float64
+# -- TEST 1: power_spectrum_1d
+n_gauss_lats = 32
+
+# Setup grid
+sinθ, wts = compute_gaussian!(n_gauss_lats)
+yarray = asin.(sinθ) .* FT(180) / π
+xarray =
+    FT(180.0) ./ n_gauss_lats * collect(FT, 1:1:(2n_gauss_lats))[:] .- FT(180.0)
+z = 1
+
+# Setup variable
+mass_weight = ones(FT, length(z));
+var_grid =
```

Comment:
What is `var_grid`?

Reply:
It is the uniform (regular) lat-long grid on which we apply the transformation to the spectral space. Perhaps I should just rename this `rll_grid` as we have used in the remapping functionality